src/components/NcAppContent/NcAppContentDetailsToggle.vue

```diff
@@ -49,14 +49,15 @@ export default {
 	},
 
 	watch: {
-		isMobile: {
-			immediate: true,
-			handler() {
-				this.toggleAppNavigationButton(this.isMobile)
-			},
+		isMobile: function() {
+			this.toggleAppNavigationButton(this.isMobile)
```

Comment:
Maybe it would make sense to use new value directly
```suggestion
		isMobile(newValue) {
			this.toggleAppNavigationButton(newValue)
```